### ISSUE TITLE
Command Router: Cannot perform [send] operation after producer has been closed

### ISSUE THREAD

**Issue by calohmn** [eclipse/hono#2746]
The following log output has appeared in a Command Router container:
````
java.lang.IllegalStateException: Cannot perform operation after producer has been closed
	at org.apache.kafka.clients.producer.KafkaProducer.throwIfProducerClosed(KafkaProducer.java:877)
	at org.apache.kafka.clients.producer.KafkaProducer.doSend(KafkaProducer.java:886)
	at org.apache.kafka.clients.producer.KafkaProducer.send(KafkaProducer.java:870)
	at io.vertx.kafka.client.producer.impl.KafkaWriteStreamImpl.lambda$send$4(KafkaWriteStreamImpl.java:92)
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$2(ContextImpl.java:313)
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Unknown Source)
````

**Comment by calohmn** [comment]
Fixed via #2751.

**Comment by calohmn** [comment]
Such errors are still getting logged on protocol adapter shutdown - see #3178.
